.github/workflows/CI.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
       - uses: mamba-org/setup-micromamba@v1
         if: ${{ matrix.rdkit-version != 'latest' }}
         with:
-          environment-file: devtools/conda-envs/CI_env.yaml
+          environment-file: environment.yaml
```

Comment:
I'm not seeing this file on main, is this correct?

Reply:
yes, i moved `devtools/conda-envs/CI_env.yaml` to `environment.yaml` in this PR to make the env usable for local installs.

Reply:
Ah, apologies, I didn't see the "moved file" thing below.